### ISSUE TITLE
Archiving wrestic repository

### ISSUE THREAD

**Issue by cimnine** [vshn/k8up#485]
## Summary

**As** developer of k8up\
**I want** to reduce the amount of dead code laying around to a minimum\
**So that** I can focus on creating value.

## Context

Since #474 is now merged, the code in the former wrestic repository, https://github.com/vshn/wrestic, is now obsolete. The repository's README.md should therefore be updateded accordingly and the repository should then be archived.

Also the description of the two Docker image repositories, hub.docker.com and quay.io, should be updated to reflect the change.

## Out of Scope

* Any changes to code.
* Removal of other artifacts.

## Further links

* https://github.com/vshn/wrestic
* https://hub.docker.com/r/vshn/wrestic
* https://quay.io/repository/vshn/wrestic?tab=info

## Acceptance criteria

* References to the former wrestic repository should be changed to point to the k8up repository.
* The README.md of the `vshn/wrestic` repository should be updated and referr the person to the k8up repository.
* Issues in `vshn/wrestic`, that are still relevant, should be copied to k8up.
* The `vshn/wrestic` repository should eventually be made read-only.

## Implementation Ideas

* Edit the README.md and push it to the `vshn/wrestic` repo.
* Search confluence, open jira tickets and the public internet for `vshn/wrestic` and update links where possible / feasible.
* Migrate still-relevant issues from wrestic repo to the k8up repo.
* Remove the Jira integration on the wrestic repository.
* Finally, make the `vshn/wrestic` repo read-only.

**Comment by tobru** [comment]
We'll do that once v2.0.0 is out and #458 is done.

**Comment by tobru** [comment]
Created a PR: https://github.com/vshn/wrestic/pull/107. We should wait until 2.0 is out before really archiving Wrestic, there are some open issues which we first have to assess, and it might be that we'll have to provide hotfixes to wrestic for some time until most installations are upgraded to 2.x.
